RN entry 2026.08.0 (#6358)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2026-okta-identity-engine/index.md

| [Prompt users to enroll a passkey is self-service EA in Preview](#prompt-users-to-enroll-a-passkey-is-self-service-ea-in-preview) | August 5, 2026 |
| [User identification policy is EA in Preview](#user-identification-policy-is-ea-in-preview) | August 5, 2026 |
| [WebAuthn authenticator enrollments include transports values is self-service EA in Preview](#webauthn-authenticator-enrollments-include-transports-values-is-self-service-ea-in-preview) | August 5, 2026 |
| [New minimum character length for AI agent names](#new-minimum-character-length-for-ai-agent-names) | August 5, 2026 |
| [New Proxy service for enhanced dynamic zones is GA in Production](#new-proxy-service-for-enhanced-dynamic-zones-is-ga-in-production) | August 5, 2026 |
| [New System Log events for Office 365 app-based provisioning is GA in Production](#new-system-log-events-for-office-365-app-based-provisioning-is-ga-in-production) | August 5, 2026 |
| [Customizable emails for Passkey (FIDO2 WebAuthn) authenticator is GA in Production](#customizable-emails-for-passkey-fido2-webauthn-authenticator-is-ga-in-production) | May 20, 2026 |

> | [Email authenticator auto-enrollment and recovery management is GA in Production](#email-authenticator-auto-enrollment-and-recovery-management-is-ga-in-production) | August 5, 2026 |

Author: ```suggestion
| [Email authenticator auto-enrollment and recovery management is GA in Production](#email-authenticator-auto-enrollment-and-recovery-management-is-ga-in-production) | May 6, 2026 |
```

| [Breached credentials protection configuration API is GA in Production](#breached-credentials-protection-configuration-api-is-ga-in-production) | August 5, 2026 |
| [Replace a Group Rule API can now update assigned groups is GA in Production](#replace-a-group-rule-api-can-now-update-assigned-groups-is-ga-in-production) | July 1, 2026 |

> | [Cross App Access support for AI agents and apps for all customers is GA in Production](#cross-app-access-support-for-ai-agents-and-apps-for-all-customers-is-ga-in-production) | August 5, 2026 |

Reviewer: ```suggestion
| [Cross App Access support for AI agents and apps for all customers](#cross-app-access-support-for-ai-agents-and-apps-for-all-customers) | August 5, 2026 |
```

| [Anything-as-a-Source device import is EA](#anything-as-a-source-device-import-is-ea) | August 5, 2026 |
| [New Research Release lifecycle](#new-research-release-lifecycle) | August 5, 2026 |
| [Advanced device posture checks is GA in Production](#advanced-device-posture-checks-is-ga-in-production) |  April 9, 2025 |

> | [Developer documentation update in 2026.08.0](#developer-documentation-update-in-2026-08-0) | August 5, 2026 |

Author: ```suggestion
| [Developer documentation updates in 2026.08.0](#developer-documentation-update-in-2026-08-0) | August 5, 2026 |
```

> | [Developer documentation updates in 2026.08.0](#developer-documentation-updates-in-2026080) | August 5, 2026 |

Reviewer 2: ```suggestion
| [Developer documentation updates in 2026.08.0](#developer-documentation-updates-in-2026-08-0) | August 5, 2026 |
```

> | [Bug fixed in 2026.08.0](#bug-fixed-in-2026080) | August 5, 2026 |

Reviewer 2: ```suggestion
| [Bugs fixed in 2026.08.0](#bugs-fixed-in-2026-08-0) | August 5, 2026 |
```

#### Replace a Group Rule API can now update assigned groups is GA in Production

 The [Replace a group rule](https://developer.okta.com/docs/api/openapi/okta-management/management/tags/grouprule/other/replacegrouprule) endpoint now supports updating the `actions` object to modify the groups assigned to a group rule. <!-- OKTA-1128862 FF EDITABLE_GROUP_RULE_TARGETS to GA Preview July 1, 2026 -->

> #### Cross App Access support for AI agents and apps for all customers is GA in Production

Reviewer: ```suggestion
#### Cross App Access support for AI agents and apps for all customers
```

#### Advanced device posture checks is GA in Production

Advanced device posture checks let admins enforce compliance based on customized device attributes that extend beyond Okta's standard checks. Using osquery, the feature facilitates real-time security assessments across macOS and Windows devices, giving orgs enhanced visibility and control over their device fleet to ensure that only trusted devices can access sensitive resources. This feature is available only if you're subscribed to *Okta Device Access (ODA)*. See [Configure advanced posture checks and custom remediation|https://developer.okta.com/docs/guides/device-assurance-posture-checks-and-remediation/main/] and the [Device Posture Checks API|https://developer.okta.com/docs/api/openapi/okta-management/management/tag/DevicePostureCheck/]. <!-- OSQUERY_CUSTOM_DEVICE_POSTURE_CHECKS_RELEASE, DEVICE_UNMANAGED_CHECKS, DEVICE_AUTHENTICATOR_INTEGRATIONS OKTA-1162878 OKTA-1193941 Preview date: April 9, 2025 -->

> #### Developer documentation update in 2026.08.0

Author: ```suggestion
#### Developer documentation updates in 2026.08.0
```

* The [Plan embedded auth app upgrades](/docs/guides/oie-upgrade-plan-embedded-upgrades/main/) guide has been reorganized. It adds a "When to use this guide" section, clarifies that Okta delivers the Sign-In Widget Gen3 only as Okta-hosted, and rewrites several sections for clarity. <!-- OKTA-1216410 -->

* The [Okta device profile](/docs/reference/okta-expression-language-in-identity-engine/#okta-device-profile) section of the Expression Language reference now documents many previously undocumented device attributes, including device.id, device.assurance.screenLockType, device.caller.* signals, and additional device.profile.* properties such as disk encryption type, integrity checks, and serial number. <!-- OKTA-1227875 -->

* The React tab of [Sign in to your SPA with Auth JS](/docs/guides/sign-in-to-spa-authjs/main/) has been rewritten to fix out-of-date sample code. It replaces the deprecated Create React App setup with Vite, updates the sign-in flow to Auth JS's Step Mode pattern, and adds the full working component code. <!-- OKTA-889035 -->

#### Bugs fixed in 2026.08.0

> * When you called the List all delegation links API (/workload-principals/api/v1/delegation-links) with a filter containing three or more `from.clientOrn` values without parentheses, an HTTP 400 `INVALID_FILTER` error was returned. (OKTA-1237936)

Reviewer 2: ```suggestion
* When you called the List all delegation links API (`/workload-principals/api/v1/delegation-links`) with a filter containing three or more `from.clientOrn` values without parentheses, an HTTP 400 `INVALID_FILTER` error was returned. (OKTA-1237936)
```

> * The [List all authorization servers for an MCP servers API](https://developer.okta.com/docs/api/secures-ai/openapi/secures-ai-resource-servers/tags/mcpserverregistration/other/listmcpserverauthorizationservers) returned a `self` link pointing to the base URL instead of the actual authorization server link. (OKTA-1236306)

Reviewer 2: ```suggestion
* The List all authorization servers for an MCP servers API (`/resource-servers/api/v1/mcp-servers/{mcpServerId}/authorization-servers) returned a `self` link pointing to the base URL instead of the actual authorization server link. (OKTA-1236306)
```

Reviewer 2: to stay the same as other bugs in this section
